decisions.md

```diff
@@ -18,18 +18,21 @@ It may not always be possible to reach a consensus. In that case, the project le
 
 ## Core Team Decisions
 
-Ultimately, the final decision on major questions is made by the Core Team. Within the Core Team, we use both consensus and voting to make decisions.
-
-Consensus is usually explicit within the Core Team, as opposed to lazy. For the kind of issues we expect the Core Team to resolve, it's important to hear from each member. The Chair is responsible for getting each member's view and determining whether there is a consensus.
+Because the Core Team generally makes major decisions, affecting multiple projects, _lazy consensus_ is normally not used. For the kind of issues we expect the Core Team to resolve, it's important to hear from each member. The Chair is responsible for getting each member's view and determining whether there is a consensus.
 
 When a consensus cannot be reached, voting is used. Additionally, issues such as those affecting the strategic direction or legal standing of the project always require a vote. The Chair is responsible for determining the necessity of a vote as well as conducting the vote.
 
 Where a vote is used, all votes, including that of the Chair are tallied. With certain exceptions noted below, a simple majority of members must vote for a proposal in order for it to pass. Note that this applies to all current members, not just those who actually vote, and therefore eliminates the need for a quorum. It is the responsibility of the Chair to collect the votes from the members, in case some of them do not respond immediately.
 
-Certain actions require a vote of two thirds of the members, rather than a simple majority.
+Since the role of the Chair is central in this process, it's important that we have a way to move forward in case of the absence or unavailability of the Chair for a particular decision or for a period of time. If the Chair will be absent for an extended period, they may appoint another Core Teamm member to serve as temporary Chair. If the Chair is unavailable and no advance provision has been made, a temporary replacement may be selected by the team. In some cases, the team may choose to proceed with one or more decisions without formally naming a replacement, in which case the team as a whole is responsible for following the process described here.
```

Comment:
This is the main addition. As suggested, I kept it fairly vague and only outlined the alternatives that could be used if the chair is unavailable for a long or short time period.

Reply:
Lgtm. My expectation: It could seem like extra effort for the team to appoint a temporary replacement if the Chair didn't. We might be more likely to want to skip that step while in the middle of work that requires a Core Team decision.

Reply:
I tried to leave both possiblilities open because I could imagine cases where you might want to do each one. For example, if there was an extremely contentious issue between two project leads, we might appoint a member who was unaffected by the dispute to serve as a moderator. I agree that we would probably want to use the lightest-weight approach in most cases.